app/helper/rtForm.php

```diff
@@ -579,6 +583,62 @@ protected function generate_textbox( $attributes ) {
 			}
 		}
 
+		/**
+		 * Generate rtmedia html inputfile in admin options.
+		 *
+		 * @access protected
+		 *
+		 * @param array $attributes Arguments to create file input control for default thumbnail generator settings.
+		 *
+		 * @return string
+		 * @throws rtFormInvalidArgumentsException Invalid argument exception.
+		 */
+		protected function generate_inputfile( $attributes ) {
+
+			$element = 'rtFile';
+			if ( is_array( $attributes ) ) {
+
+				/* Starting the input tag */
+				$html = '<input type="file" ';
+
+				/* Generating attributes */
+				$html .= $this->processAttributes( $element, $attributes );
+
+				/* Ending the tag */
+				$html .= ' /><input type="hidden" name="rtmedia-options[' . esc_attr( $attributes['name'] ) . '_hid]" value="' . esc_attr( $attributes['value'] ) . '" />';
```

Comment:
validate `$attributes['name']` and `$attributes['value']`  before using them dont trust inputs.